app/main/views/services.py

```diff
@@ -135,8 +135,13 @@ def update_service(service_id):
         service, update, update_details
     )
 
+    audit_data = {
+        'supplierName': service.supplier.name,
+        'supplierId': service.supplier.supplier_id
```

Comment:
Think this correct, the other needs to change. I'll change it now. May have knock on effects.